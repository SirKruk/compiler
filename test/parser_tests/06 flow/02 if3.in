if (p) if (q) r else s
