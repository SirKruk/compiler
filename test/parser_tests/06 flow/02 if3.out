<IF
  <ID p>
  <THEN
    <IF
      <ID q>
      <THEN
        <ID r>
        <ID s>
      >
    >
    <UNIT>
  >
>
